TypeError: Cannot convert undefined or null to object in task lint

Download 1.2 directly, then did npm install and bower install, then do "gulp serve", and here is the error output:
```
[12:37:13] Starting 'lint'...
[12:37:14] 'lint' errored after 1.18 s
[12:37:14] TypeError: Cannot convert undefined or null to object
    at Function.keys (native)
    at copyConfiguration (/Users/cnbuff410/node_modules/gulp-jscs/node_modules/jscs/lib/config/configuration.js:1181:12)
    at NodeConfiguration.Configuration._processConfig (/Users/cnbuff410/node_modules/gulp-jscs/node_modules/jscs/lib/config/configuration.js:474:5)
    at NodeConfiguration.Configuration.load (/Users/cnbuff410/node_modules/gulp-jscs/node_modules/jscs/lib/config/configuration.js:238:10)
    at StringChecker.configure (/Users/cnbuff410/node_modules/gulp-jscs/node_modules/jscs/lib/string-checker.js:67:29)
    at Checker.configure (/Users/cnbuff410/node_modules/gulp-jscs/node_modules/jscs/lib/checker.js:28:39)
    at Object.module.exports (/Users/cnbuff410/node_modules/gulp-jscs/index.js:35:10)
    at Gulp.<anonymous> (/Users/cnbuff410/gulpfile.js:124:11)
    at module.exports (/Users/cnbuff410/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/cnbuff410/node_modules/gulp/node_modules/orchestrator/index.js:273:3)
```
It's caused by this line in lint task:

  .pipe($.jscs())

I can confirm this bug. I am on OS X 10.11.1, Node 4.1.0 and NPM 3.5.0. This happens both when running `gulp` and `npm run lint`.

```bash
$ npm run lint

> @ lint /Users/ohan/path/to/project
> gulp lint

[10:47:45] Using gulpfile ~/path/to/project/gulpfile.js
[10:47:45] Starting 'lint'...
[10:47:46] 'lint' errored after 702 ms
[10:47:46] TypeError: Cannot convert undefined or null to object
    at Function.keys (native)
    at copyConfiguration (/Users/ohan/path/to/project/node_modules/jscs/lib/config/configuration.js:1181:12)
    at NodeConfiguration.Configuration._processConfig (/Users/ohan/path/to/project/node_modules/jscs/lib/config/configuration.js:474:5)
    at NodeConfiguration.Configuration.load (/Users/ohan/path/to/project/node_modules/jscs/lib/config/configuration.js:238:10)
    at StringChecker.configure (/Users/ohan/path/to/project/node_modules/jscs/lib/string-checker.js:67:29)
    at Checker.configure (/Users/ohan/path/to/project/node_modules/jscs/lib/checker.js:28:39)
    at Object.module.exports (/Users/ohan/path/to/project/node_modules/gulp-jscs/index.js:35:10)
    at Gulp.<anonymous> (/Users/ohan/path/to/project/gulpfile.js:124:11)
    at module.exports (/Users/ohan/path/to/project/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/ohan/path/to/project/node_modules/orchestrator/index.js:273:3)

npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/Cellar/node/4.1.0/bin/node" "/usr/local/bin/npm" "run" "lint"
npm ERR! node v4.1.0
npm ERR! npm  v3.5.0
npm ERR! code ELIFECYCLE
npm ERR! @ lint: `gulp lint`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ lint script 'gulp lint'.
```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -23,6 +23,8 @@ var glob = require('glob-all');
 var historyApiFallback = require('connect-history-api-fallback');
 var packageJson = require('./package.json');
 var crypto = require('crypto');
+var ensureFiles = require('./tasks/ensure-files.js');
+
 // var ghPages = require('gulp-gh-pages');
 
 var AUTOPREFIXER_BROWSERS = [
@@ -103,8 +105,19 @@ gulp.task('elements', function() {
   return styleTask('elements', ['**/*.css']);
 });
 
+// Ensure that we are not missing required files for the project
+// "dot" files are specifically tricky due to them being hidden on
+// some systems.
+gulp.task('ensureFiles', function(cb) {
+  var requiredFiles = ['.jscsrc', '.jshintrc', '.bowerrc'];
+
+  ensureFiles(requiredFiles.map(function(p) {
+    return path.join(__dirname, p);
+  }), cb);
+});
+
 // Lint JavaScript
-gulp.task('lint', function() {
+gulp.task('lint', ['ensureFiles'], function() {
   return gulp.src([
       'app/scripts/**/*.js',
       'app/elements/**/*.js',

--- a/tasks/ensure-files.js
+++ b/tasks/ensure-files.js
@@ -1,0 +1,34 @@
+var fs = require('fs');
+
+/**
+ * @param {Array<string>} files
+ * @param {Function} cb
+ */
+
+function ensureFiles(files, cb) {
+  var missingFiles = files.reduce(function(prev, filePath) {
+    var fileFound = false;
+
+    try {
+      fileFound = fs.statSync(filePath).isFile();
+    } catch (e) { }
+
+    if (!fileFound) {
+      prev.push(filePath + ' Not Found');
+    }
+
+    return prev;
+  }, []);
+
+  if (missingFiles.length) {
+    var err = new Error('Missing Required Files\n' + missingFiles.join('\n'));
+  }
+
+  if (cb) {
+    cb(err);
+  } else if (err) {
+    throw err;
+  }
+}
+
+module.exports = ensureFiles;

